src/content/app/genome-browser/components/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -15,163 +15,42 @@
  */
 
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { OutgoingActionType } from '@ensembl/ensembl-genome-browser';
+import { useSelector } from 'react-redux';
+
+import RadioGroup, {
+  RadioOptions
+} from 'src/shared/components/radio-group/RadioGroup';
+import GeneTrackConfig from './track-config-views/GeneTrackConfig';
+import RegularTrackConfig from './track-config-views/RegularTrackConfig';
 
 import {
-  updateTrackConfigNames,
-  updateTrackConfigLabel,
-  updateApplyToAll,
-  updateApplyToAllTrackLabels,
-  updateApplyToAllTrackNames
+  getTrackType,
+  TrackType
 } from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
 
 import {
-  getTrackConfigNames,
-  getTrackConfigLabel,
   getApplyToAllConfig,
-  getBrowserCogTrackList,
   getBrowserSelectedCog
-} from 'src/content/app/genome-browser/state/track-config/trackConfigSelectors';
-
-import analyticsTracking from 'src/services/analytics-service';
+} from '../../state/track-config/trackConfigSelectors';
 
-import useGenomeBrowser from 'src/content/app/genome-browser/hooks/useGenomeBrowser';
-
-import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
-import RadioGroup, {
-  OptionValue,
-  RadioOptions
-} from 'src/shared/components/radio-group/RadioGroup';
+import useBrowserTrackConfig from './useBrowserTrackConfig';
 
 import styles from './BrowserTrackConfig.scss';
 
+const getTrackConfigComponent = (trackType: TrackType) => {
+  switch (trackType) {
+    case TrackType.GENE:
+      return <GeneTrackConfig />;
+    case TrackType.REGULAR:
+      return <RegularTrackConfig />;
+  }
+};
+
 export const BrowserTrackConfig = () => {
   const applyToAllConfig = useSelector(getApplyToAllConfig);
-  const browserCogTrackList = useSelector(getBrowserCogTrackList);
   const selectedCog = useSelector(getBrowserSelectedCog) || '';
```

Comment:
I've mentioned about giving a default empty string value below. Please see that comment.